### PR TITLE
dev-utils/pycharm-professional: version bump

### PR DESCRIPTION
**Pull request by switch87**
Pycharm Professional 4.5.4
@xmw , Also look at #26 please.

### CODE REVIEW

**Comment by switch87** [comment]
what have errors for mithtv to do with my pull request?

**Comment by hasufell** [comment]
> Please note that the issues may come from the underlying Gentoo repository state rather than the pull request itself.

**Comment by switch87** [comment]
And what do I have to do to resolve the conflicts....

**Comment by mgorny** [comment]
Pull changes from Gentoo master, and rebase against the new master.

**Comment by mgorny** [comment]
@xmw

**Comment by switch87** [comment]
ok, I did a rebase

**Comment by switch87** [comment]
@mgorny I don't think @xmw is going to take a look at this (as he neither did to #26 )

File: dev-util/pycharm-professional/pycharm-professional-4.5.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	opt/${PN}/bin/fsnotifier64
+	opt/${PN}/bin/libyjpagent-linux.so
+	opt/${PN}/bin/libyjpagent-linux64.so"
+
+MY_PN=${PN/-professional/}
+S="${WORKDIR}/${MY_PN}-${PV}"
+
+src_install() {
+	insinto /opt/${PN}
+	doins -r *
```
Comment (mgorny) [inline dev-util/pycharm-professional/pycharm-professional-4.5.4.ebuild]: Better to do `doins  -r .`. Avoids unnecessary pattern expansion.

**Comment by switch87** [comment]
@mgorny I changed it

**Comment by mgorny** [comment]
It seems that you need to rebase again. Would you like to become a proxied maintainer for pycharm-professional? You'd get bugmail and be responsible for fixing stuff.

**Comment by switch87** [comment]
Yes I know, but I am on a Windows pc at the moment and for some reason rebasing will not work on it, I ame trying it for the past 15 minutes :-1: ... I will also change the xml to make me proxied maintainer if I am back on linux.

**Comment by switch87** [comment]
@mgorny I did a rebase and added myself as proxied maintainer

**Comment by xmw** [comment]
Fixed with https://cgit.gentoo.org/repo/gentoo.git/commit/dev-util/pycharm-professional/pycharm-professional-4.5.4.ebuild?id=74a3e13cd7a66dfdfc9481a8717d69511b244a51

No need for proxy setup, just file bugzilla reports to notify me.

**Pull request closed without merging**